urgent-care-api/pom.xml

```diff
@@ -103,6 +103,30 @@
     </plugins>
   </build>
   <dependencies>
+    <!-- https://mvnrepository.com/artifact/javax.persistence/javax.persistence-api -->
+    <dependency>
+      <groupId>javax.persistence</groupId>
+      <artifactId>javax.persistence-api</artifactId>
+      <version>2.2</version>
```

Comment:
Should this be a property?

Reply:
After discussion, don't think that it's necessary to pull it out into a property.